kie-server-parent/kie-server-services/kie-server-services-openshift/src/test/java/org/kie/server/services/openshift/impl/storage/cloud/KieServerStateOpenShiftRepositoryEnhancedTest.java

```diff
@@ -0,0 +1,24 @@
+package org.kie.server.services.openshift.impl.storage.cloud;
+
+import org.junit.Test;
+import org.kie.server.api.KieServerConstants;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+public class KieServerStateOpenShiftRepositoryEnhancedTest extends KieServerStateOpenShiftRepositoryTest {
+
+    @Test(expected = IllegalStateException.class)
+    public void testLoadWithNoInitCMandMissMatchedServerId() {
+        assertNotNull(client.configMaps().withName(TEST_KIE_SERVER_ID).delete());
+        System.setProperty(KieServerConstants.KIE_SERVER_ID, TEST_KIE_SERVER_ID);
+        assertEquals(TEST_KIE_SERVER_ID, repo.load(TEST_KIE_SERVER_ID)
+                     .getConfiguration().getConfigItemValue(KieServerConstants.KIE_SERVER_ID));
+    
+        System.setProperty(KieServerConstants.KIE_SERVER_ID, "wrongId");
+        repo.load(TEST_KIE_SERVER_ID)
+            .getConfiguration().getConfigItemValue(KieServerConstants.KIE_SERVER_ID);
+        System.clearProperty(KieServerConstants.KIE_SERVER_ID);
```

Comment:
This line is never called, please put it to `@After`

Reply:
Moved to @After method.